lib/utils/tests-options-validator.js

```diff
@@ -258,6 +258,14 @@ module.exports = class TestsOptionsValidator {
    * @return {boolean}
    */
   validateParallel() {
+    const parallelValue = parseInt(this.options.parallel);
```

Comment:
Generally speaking, you should always pass a radix to `parseInt` (Note the radix does **not** default to 10!!!! [see MDN page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt)).

```suggestion
    const parallelValue = parseInt(this.options.parallel, 10);
```

Reply:
Thanks @rwjblue !! Fixed!